# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../findutils/gnulib-wchar.patch.cross
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./gnulib/lib/wchar.in.h.orig	2007-12-02 13:48:45.000000000 +0100
+++ ./gnulib/lib/wchar.in.h	2008-03-28 13:06:12.000000000 +0100
@@ -27,7 +27,7 @@
  */
 
 #ifndef _GL_WCHAR_H
-
+#if 0
 /* Tru64 with Desktop Toolkit C has a bug: <stdio.h> must be included before
    <wchar.h>.
    BSD/OS 4.0.1 has a bug: <stddef.h>, <stdio.h> and <time.h> must be
@@ -35,7 +35,7 @@
 #include <stddef.h>
 #include <stdio.h>
 #include <time.h>
-
+#endif
 /* Include the original <wchar.h> if it exists.
    Some builds of uClibc lack it.  */
 /* The include_next requires a split double-inclusion guard.  */
@@ -44,7 +44,7 @@
 #endif
 
 #ifndef _GL_WCHAR_H
-#define _GL_WCHAR_H
+/*#define _GL_WCHAR_H*/
 
 /* The definition of GL_LINK_WARNING is copied here.  */
 
